flycheck.el

```diff
@@ -9547,7 +9587,12 @@ See URL `https://eslint.org/'."
        (flycheck-verification-result-new
         :label "config file"
         :message (if have-config "found" "missing or incorrect")
-        :face (if have-config 'success '(bold error)))))))
+        :face (if have-config 'success '(bold error))))))
+  :error-explainer
+  (lambda (err)
+    (let ((error-code (flycheck-error-id err))
+          (url "https://eslint.org/docs/rules/%s"))
+      (and error-code `(url . ,(format url error-code))))))
```

Comment:
I think we should skip non-builtin rules:
```suggestion
      (unless (cl-find ?/ error-code) ; skip non-builtin rules
        `(url . ,(format url error-code))))))
```

Reply:
Good point, committed.

Reply:
I did keep the `error-code` test though, because we don't want to display a page when the error code is nil.